documentation/changelogs/catalog/2023.04.27.02.43.13.md

```diff
@@ -0,0 +1,64 @@
+        # 2023.04.27.02.43.13
+
+        ## Improvements
+
+- Add `LABEL` to link repo with GHCR image (#1223) @dhruvkb
```

Comment:
Oh dear, that indentation seems off 😰

Reply:
I wonder if we need to pass the substrings to dedent: https://github.com/WordPress/openverse/blob/f5c6c7ad9a34c25c8a7afafcee7b95b34686e625/automations/python/workflows/write_changelog.py#L12-L18

Maybe something about pulling it from the release draft text has them indented strangely?

Reply:
Pushed https://github.com/WordPress/openverse/pull/1914/commits/ff73f214f1a94905ca4876af8b6522674e5aded4 to fix for this file and made a change to the write script to hopefully prevent it in the future. Also opened #1916 to help avoid problems like these.